BackBLAST

```diff
@@ -474,16 +494,50 @@ function main() {
     printf "${SCRIPT_NAME}: pipeline to search for and visualize gene homologs across multiple genomes.\n"
     printf "Copyright Lee H. Bergstrand and Jackson M. Tsuji, Neufeld Research Group, 2019\n"
     printf "Version: ${VERSION}\n\n"
-    printf "Please specify a run mode for further usage instructions:\n"
+    printf "Please specify a run mode for the main workflow for further usage instructions:\n"
     printf "   1. setup: for setting up pre-run configuration files\n"
     printf "   2. run: to start a run using configuration files\n"
     printf "   3. auto: to skip setup and run the pipeline end-to-end with default values\n\n"
+    printf "Or run a specific step in the workflow manually:\n"
```

Comment:
If the user wants to call one of the utility scripts (rather than run the full pipeline), they can now do so directly through the main `BackBLAST` script here.